wait-for-mysql.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash -e
+
+while ! db dockerized-mysql </dev/null
+do
+  >&2 echo "Waiting 5 seconds for MySQL to be available"
+  sleep 5
+  nc -vz mysqldb 3306 || true # TODO TEMP
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker